tests_func/utils/util.go

```diff
@@ -0,0 +1,183 @@
+package utils
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+)
+
+func MapEnvToListEnv(mpEnv map[string]string) []string {
+	var result []string
+	for key, value := range mpEnv {
+		result = append(result, fmt.Sprintf("%s=%s", key, value))
+	}
+	return result
+}
+
+func MergeEnvs(env []string, values []string) []string {
+	envMap := make(map[string]string, 0)
+	for _, line := range append(env, values...) {
+		name, value := SplitEnvLine(line)
+		envMap[name] = value
+	}
+	updatedEnv := make([]string, 0)
+	for name, value := range envMap {
+		updatedEnv = append(updatedEnv, fmt.Sprintf("%s=%s", name, value))
+	}
+	return updatedEnv
+}
+
+func StringInSlice(a string, list []string) bool {
+	for _, b := range list {
+		if b == a {
+			return true
+		}
+	}
+	return false
+}
+
+func SplitEnvLine(line string) (string, string) {
```

Comment:
with github.com/heetch/confita all this code will be unnecessary, right?

Reply:
I am really not sure about this. Afaik github.com/heetch/confita parses config structs into another structs and it's quite complex for this case I think as here I initialize values individually without combination of different configs, also I am trying to use list of strings of format "%s=%s" like env vars to be able to easily pass it to exec cmd.